src/Framework/Runner/Runner.cs

```diff
@@ -440,6 +451,19 @@ public void SetupTests()
                     }
                 }
             }
+            if (CopyUserAddins)
+            {
+                var files = Directory.GetFiles(GetRevitUserAddinFolder());
+                foreach (var file in files)
+                {
+                    if (file.EndsWith(".addin", StringComparison.OrdinalIgnoreCase))
```

Comment:
This logic is duplicated from the part above where we copy the addins. In the spirit of leaving the code cleaner than you found it, can you refactor this copy logic into a separate method and replace it here and above?